### ISSUE TITLE
PRM: Assign constructions not based on standards type

### ISSUE THREAD

**Issue by jmarrec** [NREL/openstudio-standards#46]
@asparke2 

I'm not sure I'm a big fan of using the standards information attached to the construction that is applied to a surface to determine what should be the baseline construction of this surface.

It will not even work for most of the current Construction Sets that ship with OS, since for example Ground Walls, Floors, Ceilings and Exterior Floors can refer to the same construction

OpenStudio is smart enough as it is to correctly classify surfaces based on their boundary condition, z coordinate, and tilt. I think we should rely on that rather than rely on people creating on construction for each surface type AND filling out the standards information on each on them.

**Comment by asparke2** [comment]
Here's how the code works now:

1. `Standards.Model.apply_performance_rating_method_construction_types` changes all of `standardsConstructionType` for Constructions used on Exterior Walls to SteelFramed, Roofs to IEAD, Floors to SteelFramed, Slabs to Unheated, and Basement Walls to Unheated.
2. `Standards.Model.apply_standard_constructions` modifies the constructions used for each of specified types of surface/boundary condition pairs to have the correct thermal properties per the `standardsConstructionType` field.  For walls, roofs, etc, this was set by the previous step.  For windows, it will use whatever user specified in the proposed model.

I set it up this way because for some codes other than 90.1 (India's ECBC, for example), the baseline construction types follow the proposed.  This means the user needs a way to specify them, and the `standardsConstructionType` seemed like a good place to specify.  So for that code, you'd simply skip step 1 and go straight to step 2.

As you pointed out, this isn't ideal because a single construction could potentially be used on two surface types like Floors and Walls, but I figured this would be uncommon.

I agree that we should use OS for determining type of surface (wall, floor, etc) instead of using the `IntendedSurfaceType` field of the construction.

With this backstory, does the current approach still make sense?  I'm open to better ideas because I didn't really like this one but couldn't think of a better approach that didn't involve creating lots of duplicate constructions.
